tests/AMSlib/CMakeLists.txt

```diff
@@ -155,8 +155,9 @@ function(ADDTEST exe test_name)
   endif()
 endfunction()
 
-BUILD_TEST(ams_allocator_test ams_allocate.cpp)
-ADDTEST(ams_allocator_test AMSAllocate)
+# This test requires Allocate
```

Comment:
dead code? are those tests going to be replaced?

Reply:
Fixed.